README.md

```diff
@@ -6,15 +6,21 @@ intended to provide an overview; For full documentation please check the manual
 
 ## Contents
 
-1. [Introduction](#introduction)
-2. [Configuring services](#configuring-services)
+1. [Getting started guide](#getting-started)
+2. [Introduction](#introduction)
+3. [Configuring services](#configuring-services)
     1. [Service types](#service-types)
     2. [Service description files](#service-description-files)
-3. [Running Dinit](#running-dinit)
-4. [Controlling services](#controlling-services)
+4. [Running Dinit](#running-dinit)
+5. [Controlling services](#controlling-services)
     1. [Service hierarchy and states](#service-hierarchy-and-states)
     2. [Using dinitctl](#using-dinitctl)
 
+## Getting started
+
+The impatient may wish to check out the
```

Comment:
Maybe move this in to the preamble paragraph, i.e. "This is the README for Dinit, the service manager and init system. It is intended to provide an overview; For full documentation please check the manual pages. The impatient may wish to read the Getting Started guide."

(To me it feels like it doesn't need a section to itself. Or if it has its own section, that should be after the Introduction rather than before it).